src/app/api/test-types/model/testType.ts

```diff
@@ -80,14 +80,15 @@ export interface TestType {
     forVehicleWheels?: Array<number>;
 }
 export namespace TestType {
-    export type ForVehicleTypeEnum = 'psv' | 'hgv' | 'trl' | 'car' | 'lgv' | 'motorcycle';
+    export type ForVehicleTypeEnum = 'psv' | 'hgv' | 'trl' | 'car' | 'lgv' | 'motorcycle' | 'small trailer';
```

Comment:
I think we're classifying trailers as small trailers by their EU vehicle category, and not a new vehicle type

Reply:
Again that isn't what's on the documentation, but I will look into this.